### PR TITLE
When a command fails, print its error in multiple lines

### PR DESCRIPTION
**Pull request by querti**
When skopeo copy fails, the error message cannot be accessed because the log of the command output gets truncated. Since the command generates new line characters, use them to split the output and log each line separately.

Refers to CLOUDDST-19503

### CODE REVIEW

**Review by zxiong — APPROVED** [review]

File: pubtools/_quay/command_executor.py [hunk 1]
```diff
@@ -182,7 +182,9 @@ def _run_cmd(self, cmd, err_msg=None, tolerate_err=False, stdin=None):
         out, err = p.communicate(input=stdin)
 
         if p.returncode != 0 and not tolerate_err:
-            LOG.error("Command {0} failed with {1}".format(cmd, err))
+            LOG.error("Command {0} failed with the following error:".format(cmd))
```
Comment (emilyzheng) [inline pubtools/_quay/command_executor.py:185]: Without this change, error can already be shown in multiple lines if it includes "\n": 

```
err = "multiline\nerror\nmessage"

2023-08-01 02:01:30 +0000 [ERROR   ] Command skopeo copy --all docker://registry-proxy.engineering.redhat.com/rh-osbs/rhel8-redis-5:rhel-8.6.0-z-containers-candidate-36737-20220620193816 docker://quay.io/redhat-dev2/test----repo1:121 --format v2s2 failed with multiline
error
message
```
Does error really have "\n"? Seems only stdout has it.

File: pubtools/_quay/command_executor.py [hunk 2]
```diff
@@ -182,7 +182,9 @@ def _run_cmd(self, cmd, err_msg=None, tolerate_err=False, stdin=None):
...
+            for line in err.splitlines():
+                LOG.error(line)
```
Comment (midnightercz) [inline pubtools/_quay/command_executor.py:187]: Can you add an indentation here? For example "    " could be enough to make it more readable in logs

**Review by midnightercz — APPROVED** [review]

File: pubtools/_quay/command_executor.py (hunk 1 above)
Comment (querti) [reply pubtools/_quay/command_executor.py:186]: You're right, looks like splitting by \n is not the correct solution for local executor. I have instead added textwrap that splits the long message based on length.

File: pubtools/_quay/command_executor.py (hunk 2 above)
Comment (querti) [reply pubtools/_quay/command_executor.py]: Added.

**Review by emilyzheng — APPROVED** [review]

**Pull request merged**